src/help/faq/README.md

```diff
@@ -25,7 +25,14 @@ If you're having problems with the app, please check here before posting. If you
 ---
 
 ### App crashes on launch
-If you are using the **Public build**, check if you are on iOS 13.4+ or iPadOS 13.4+. If you are using the **Beta / Patron build**, ping @Paper on the Discord #supporter channel.
+If you are using the **Public build**, check if you are on iOS 13.4+ or iPadOS 13.4+. If you are using the **Beta / Patron build**, ping @Paper on the Discord #supporter-chat channel.
+
+---
+
+### Both Paperback and AltStore crash on launch
+Applications installed with AltStore need to be refreshed every seven days. While AltServer and Mail or iCloud/iTune are open and your device is connected to the same network as your computer, AltStore will try to refresh the app automatically.
```

Comment:
```suggestion
Applications installed with AltStore need to be refreshed every seven days. Ensure that AltServer and Mail or iCloud/iTunes are open, and your device is connected to the same network as your computer. AltStore will try to refresh the app automatically.
```

Optionally: Elaborate a little more on what you should do if AltStore doesn't refresh automatically

```suggestion
Applications installed with AltStore need to be refreshed every seven days. Ensure that AltServer and Mail or iCloud/iTunes are open, and your device is connected to the same network as your computer. AltStore will try to refresh the app automatically. If AltStore does not refresh automatically, you may trigger this process manually by plugging your device into a computer running AltStore/Itunes, and then running the refresh on your device through the app.
```

Reply:
Good idea. Maybe we should explain that people can refresh from AltStore app. Or does it seams obvious?

And a small thing that could be improved is what should run on the computer: AltServer and iTune/iCloud or Mail.
```suggestion
Applications installed with AltStore need to be refreshed every seven days. Ensure that AltServer and Mail or iCloud/iTunes are open, and your device is connected to the same network as your computer. AltStore will try to refresh the app automatically. If AltStore does not refresh automatically, you may trigger this process manually by plugging your device into a computer running AltServer and Mail or iCloud/Itunes, and then running the refresh on your device through AltStore app.
```
Or maybe we said enough that iTune/iCloud or Mail have to be open:
```suggestion
Applications installed with AltStore need to be refreshed every seven days. Ensure that AltServer and Mail or iCloud/iTunes are open, and your device is connected to the same network as your computer. AltStore will try to refresh the app automatically. If AltStore does not refresh automatically, you may trigger this process manually by plugging your device into a computer running AltServer, and then running the refresh on your device through AltStore app.
```

Reply:
I think your second suggestion is great!